tests/migrations/test_operations.py

```diff
@@ -5929,6 +5929,41 @@ def test_add_field_after_generated_field(self):
         self.assertEqual(pony_new.generated, 1)
         self.assertEqual(pony_new.static, 2)
 
+    def test_change_id_pk_to_uuid_pk(self):
+        app_label = "test_change_id_pk_to_uuid_pk"
+        project_state = self.set_up_test_model(app_label)
+        operation_1 = migrations.AddField(
```

Comment:
This test would be more readable with meaningful variable names. I'd also move them closer to where they are used.